### ISSUE TITLE
"Get any directory online" does not work

### ISSUE THREAD

**Issue by cogdog** [sintaxi/surge#107]
I've tried the suggestion of surging a directory of images (as well as another of gifs), but the generated site always results in not found, e.g. http://gifdog.surge.sh

**Comment by danreeves** [comment]
Sounds like you don't have either an `index.html` or a `200.html` page. If you add one of these it'll stop saying "page not found". 

If you try going directly to one of your images (e.g. http://gifdog.surge.sh/pathto/image.gif) it should work.

**Comment by kennethormandy** [comment]
Yes, that will be the case, thanks @danreeves. We’re planning on improving the explanation on the default 404 page, maybe even doing something especially for this situation where there are other files but not an `index.html` file. Thanks for the feedback!

**Comment by cogdog** [comment]
Thanks, I pretty much guessed that. The way its written (to me), it sounds like you could share a directory of media like apache generates a directory view. 

I'm a fan, been trying to get some education folks to consider what surge could do as a way for students to create public sites of their own work http://cogdogblog.com/2015/05/30/with-surge/
